### ISSUE TITLE
Receiver data uploads missing GPS fixes

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#482]
There are several instances where data uploads show GPS fixes occur for a few days before cutting out while antennas remain active. Initially, this appears to be related to a known bug with older USB GPS modules, but upon further inspection it's not that clear any more. For days where GPS fixes are not indicated in output plots, we still have several readings in the raw data.

Take a look at the following timeline:
https://motus.org/data/receiver/timeline?id=2996&year=2019

Last detection data are from April 15, but last activity info is from April 21. GPS fixes are only displayed up until April 2 according to timeline, but GPS fixes can be found in text files for all days including Apr 21.

I looked through several of the data files from this receiver for around April 1-21 and all of them had 12 GPS fixes every hour. Why is this not showing up on the timeline? Do tag detections from these periods also lack GPSlat/GPSlon?

This is the same receiver in question for issue #481, but I don't think it's related since GPS fixes are missing from a different date and it appears this is occurring at several receivers, not just this one. For example, Martinique (deployment 5151, device 643): 
https://motus.org/data/receiver/timeline?id=2724&year=2019

**Comment by Hussell** [comment]
The tag finder has some code to check for a "stuck" GPS: identical lats and longs down to the last decimal place. Checking the data files, I see the GPS reading did indeed get stuck partway through April 2nd.

I've never run into this before, so I don't know why or how the GPS gets stuck, or what to do about it. Stay tuned.

**Comment by Hussell** [comment]
The timestamp gets stuck too, on a value with a 3 decimal places, where all previous timestamps are rounded to the nearest second. So I think there's a bug on the SensorGnome. Do you have any examples that are _not_ from early this April? If all examples are from early April then this might be related to the recent GPS week rollover event.

**Comment by leberrigan** [comment]
Nice catch - I missed the fact they all had identical timestamps. I believe this is related to stuck USB GPS modules and nothing more... This is the reason we switched to using GPS HATs with antennas. Perhaps there's a way to fix this with an update to the Sensorgnome firmware, but this hasn't been worked on as far as I can tell.

I have looked at a few other receivers with data from before well before April (into 2014) and got the same results so that rules out GPS rollover being the problem.
